eve/main.yml

```diff
@@ -24,6 +50,11 @@ stages:
         namespaces:
         - "testNamespace"    # <<< the default namespace for that stage
     steps:
+    - ShellCommand:
+        name: Install curl
+        command: |-
+          apk --update upgrade
+          apk --update add curl
```

Comment:
this must be cached in the worker definition to avoid the extra internet dependency (flakiness control)

Reply:
There's a pull request regarding https://github.com/scality/zenko-releng/pull/2

Waiting for merge and approvals